tools/topology/sof/pipe-ca-dcblock-playback.m4

```diff
@@ -0,0 +1,102 @@
+# Low Latency Passthrough Pipeline and PCM
```

Comment:
How should we manage the pipeline topology files while we have more codecs of Codec Adapter?
I feel like this file is mostly similar to pipe-processing-playback.m4, except the part of control bytes. So we should probably create a common pipe-codec-adapter-playback.m4 file and leave the part of control bytes empty (or make a dummy default). Then each codec requires to add its own file to define the own control bytes.
WDYT?

Reply:
ack - common codec pipe a good idea.

Reply:
I have uploaded a PR for this https://github.com/thesofproject/sof/pull/3736
Please help to review it, thanks.

Reply:
I think we have one for playback, but I think one for input would be a good addition, mind adding that?

Reply:
@johnylin76, if you have more codecs then you should add more adapter. So, assuming your pipeline consists of 2 different codecs it should looks like this (assuming playback and no other processing components) HOST -> CODEC_ADAPTER_1 -> CODEC_ADAPTER_2 -> DAI

We have a common, example pipeline for playback already -> pipe-processing-playback.m4